src/platform/cocoamain.mm

```diff
@@ -1218,6 +1218,227 @@ - (IBAction)preferences:(id)sender {
     [NSApp stop:nil];
 }
 
+#ifdef HAVE_SPACEWARE
```

Comment:
Yes, exactly, you have an if in CMakeLists to *not* link to `${SPACEWARE_LIBRARIES}` on Apple platforms.